instance_configs.go

```diff
@@ -78,26 +72,26 @@ type InstanceConfigsPagedResponse struct {
 
 // InstanceConfigCreateOptions are InstanceConfig settings that can be used at creation
 type InstanceConfigCreateOptions struct {
-	Label       string                    `json:"label,omitempty"`
-	Comments    string                    `json:"comments,omitempty"`
-	Devices     InstanceConfigDeviceMap   `json:"devices"`
-	Helpers     *InstanceConfigHelpers    `json:"helpers,omitempty"`
-	Interfaces  []InstanceConfigInterface `json:"interfaces"`
-	MemoryLimit int                       `json:"memory_limit,omitempty"`
-	Kernel      string                    `json:"kernel,omitempty"`
-	InitRD      int                       `json:"init_rd,omitempty"`
-	RootDevice  *string                   `json:"root_device,omitempty"`
-	RunLevel    string                    `json:"run_level,omitempty"`
-	VirtMode    string                    `json:"virt_mode,omitempty"`
+	Label       string                                 `json:"label,omitempty"`
+	Comments    string                                 `json:"comments,omitempty"`
+	Devices     InstanceConfigDeviceMap                `json:"devices"`
+	Helpers     *InstanceConfigHelpers                 `json:"helpers,omitempty"`
+	Interfaces  []InstanceConfigInterfaceCreateOptions `json:"interfaces"`
```

Comment:
Should we put a note somewhere as a reminder for this breaking change?

Reply:
I think we can write that in the future release note?

Reply:
Gotcha. Maybe also add a note in this PR description so it's easier to track?